src/posts/lucky_0.30.cr

```diff
@@ -0,0 +1,215 @@
+    * cross-site scripting
+    * SQL injection
+    * header tampering
+    * and more...
+
+    Bright has an API that allows you to scan your application for different vulnerabilities
+    using their [SecTester](https://github.com/NeuraLegion/sec_tester) shard. By signing up
+    for an API key, you can test specific pages of your application with CI which will help
+    prevent pushing broken code to production.
+
+    We have created a thin wrapper ([LuckySecTester](https://github.com/luckyframework/lucky_sec_tester))
+    which makes writing some of these tests with your Lucky application a little easier. For now,
+    this feature is opt-in when generating a new Lucky app, but it can be added to any existing app
+    fairly easily.
+
+    To generate a new app with the SecTester, just run `lucky init.custom your_app_name --with-sec-test`.
+    This will generate a new application with the LuckySecTester setup and ready to go, along with several
+    specs already written against the existing code. This will help to serve as a template for what you
+    can add. This will also generate a Github CI already setup to run with Bright!
+
+    Due to the time it takes to run these specs, we've placed the security specs in their own spec
+    file, and behind a compile-time flag `crystal spec -Dwith_sec_tests`. This gives you the option to
+    run them only when you really need them. (e.g. during CI).
+
+    Following the Lucky mantra of catching bugs in development, it is our hope that this new integration
+    gives you peace of mind that your production app is a lot safer and solid.
+
+    ### Faster specs
+
+    After quite a bit of refactoring, we now have database related specs running in a transaction
+    instead of truncating between each spec. This has shown to give at least 4x speed to your specs.
+
+    To enable this, you'll need to update a few things.
+
+    * Remove the `spec/setup/clean_database.cr` file.
+    * Update your `spec/spec_helper.cr` file with this line:
+
+    ```crystal
+    Avram::SpecHelper.use_transactional_specs(AppDatabase)
+    ```
+
+    Adding to the speed, action specs are faster as well! This is done by building the context
+    internally instead of making the full HTTP request to the Action. In your `ApiClient`, you
+    just need to add this line:
+
+    ```crystal
+    class ApiClient < Lucky::BaseHTTPClient
+      # Add this line
+      app AppServer.new
+
+      # ...
+    end
+    ```
+
+    Finally, if you want to try out even more speed for your Flow specs, you can upgrade to v0.8.0.
+    This release of LuckyFlow wasn't included for this 0.30.0 release of Lucky, but you can still include
+    it to test it out.
+
+    LuckyFlow 0.8.0 brings in a new Driver registration along with a new "Webless" driver. By using this
+    driver, no Chrome process is booted which saves on both CPU and memory. The caveat to this is the
+    limitations to any javascript related interactions.
```

Comment:
```suggestion
    driver, no Chrome process is booted which saves on both CPU and memory. The caveat to this is that
    JavaScript is not enabled so those interactions will not function.
```

Is this correct? I wasn't sure what you meant by "the limitations to any javascript related interactions"

Reply:
Exactly.